spec/lib/errors/rescue_development_errors_spec.rb

```
@@ -39,7 +39,7 @@ def index
 
     it 'renders an explanation with helpful advice' do
```

This is no longer accurate.

---

Instead of the mock index above that raises the error, you could do a before { Campaign.destroy_all }, then visit the normal index page, and expect the (newly created) default campaign to show up.

---

Sure. Will work on it.